include/cuco/static_map.cuh

```diff
@@ -1414,6 +1414,8 @@ class static_map {
                                sentinel::erased_key<Key>{erased_key_sentinel_});
   }
 
+  atomic_ctr_type* get_num_successes() const noexcept { return num_successes_; }
+
```

Comment:
This shouldn't be public. `num_successes_` is for internal use only and it's reset to zero each time before using.